packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -44,39 +38,22 @@ const DECODE_CONTRACT_CALL_RESULT_ABI = [
   },
 ];
 
-// TODO: get these from terms
-const AAVE_POOL_ADDRESS = '0xccEa5C65f6d4F465B71501418b88FBe4e7071283';
-const USDC_TOKEN_ADDRESS = '0xCaC7Ffa82c0f43EBB0FC11FCd32123EcA46626cf'; // not circle USDC
-
-export const supportedEVMChains: CaipChainId[] = [
-  'eip155:43114', // Avalanche
-  'eip155:8453', // Base
-  'eip155:1', // Ethereum
-];
-
 const TypeShape = M.or(...keys(YieldProtocol));
-export const ChainShape = M.or(...values(PositionChain));
 const PositionShape = M.splitRecord({}); // TODO
+const VowStringShape = M.any(); // Vow(M.string())
 
 const KeeperI = M.interface('keeper', {
-  add: M.call(
-    TypeShape,
-    M.or(ChainShape),
-    M.remotable('OrchestrationAccount'),
-  ).returns(M.number()),
-  getPositions: M.call(TypeShape, ChainShape).returns(M.arrayOf(PositionShape)),
-  getAccount: M.call(M.number(), TypeShape).returns(
+  addAavePosition: M.call(M.string()).returns(),
+  addCompoundPosition: M.call(M.string()).returns(),
+  addUSDNPosition: M.call(
+    M.string(),
     M.remotable('OrchestrationAccount'),
-  ),
-  getRemoteAccountAddress: M.call(M.number()).returns(
-    M.or(M.string(), M.undefined()),
-  ),
-  sendGmp: M.call(M.remotable('Seat'), M.record()).returns(M.promise()),
-});
-
-const HolderI = M.interface('Holder', {
-  supplyToAave: M.call(M.remotable('Seat')).returns(M.promise()),
-  withdrawFromAave: M.call(M.remotable('Seat')).returns(M.promise()),
+  ).returns(),
+  getPositions: M.call(TypeShape, M.string()).returns(M.arrayOf(PositionShape)),
+  getAccount: M.call(TypeShape).returns(M.remotable('OrchestrationAccount')),
+  initGmp: M.call(M.remotable('OrchestrationAccount')).returns(),
+  getRemoteAddress: M.call().returns(VowStringShape),
+  wait: M.call(M.bigint()).returns(VowShape),
 });
 
 const EvmTapI = M.interface('EvmTap', {
```

Comment:
I think the name `EvmTapI` is misleading; we don't get to define this interface.